.github/workflows/nixos-manual.yml

```diff
@@ -0,0 +1,20 @@
+name: NixOS manual checks
+
+on:
+  pull_request:
```

Comment:
Just an FYI as actions have changed since the PR was opened: this won't run automatically for first time contributors so it's possible that the docs could drift out of sync.

https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/

Would need to do the same as we did in https://github.com/NixOS/nixpkgs/commit/5ba2e4e9c65d32ea3d7def475655227a3835d2dc (`pull_request_target`, `permissions:`, and the PR merge ref) to make it run unconditionally.

Reply:
ah, great point.